.travis.yml

```diff
@@ -1,5 +1,21 @@
 language: haskell
 
+ghc: 7.8
+
+addons:
+  apt:
+    packages:
+    - ocaml
+    - camlidl
+
+sudo: false
+
+cache:
+  apt: true
+  directories:
```

Comment:
Wiping the cache when there's a conflict sounds fine to me. It's still a net win for us and travis :)

Sent from my iPhone

> On Apr 19, 2015, at 23:31, Michael Smith <notifications@github.com> wrote:
> 
> In .travis.yml:
> 
> > @@ -1,5 +1,21 @@
> >  language: haskell
> >  
> > +ghc: 7.8
> > +
> > +addons:
> > +  apt:
> > +    packages:
> > +    - ocaml
> > +    - camlidl
> > +
> > +sudo: false
> > +
> > +cache:
> > +  apt: true
> > +  directories:
> I know Halcyon handles this well, but I don't know about Cabal. It's hacky, but the Travis script could throw away the cache and reinstall if Cabal says there will be a conflict.
> 
> —
> Reply to this email directly or view it on GitHub.
>